test/lib/mayaUsd/render/mayaToHydra/cpp/testFlowViewportAPIViewportInformation.py

```diff
@@ -40,6 +44,10 @@ def test_RendererSwitching(self):
             cmds.mayaHydraCppTest(f="FlowViewportAPI.viewportInformationWithHydraAgain")
             
     def test_MultipleViewports(self):
+
+        # reset all default viewports for VP2 as the test framework sets it to Storm in base class    
+        self.setViewport2Renderer()
```

Comment:
Why did you have to set this line ? I want to be sure we use Storm for the tests to be meaningful...

Reply:
since we are setting Storm as the default renderer earlier in our unit test setup, and this unit test expects no Storm viewport, explicitly setting here so that the CPP tests pass.